heroku.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9
+
+# Install dependencies
+RUN apt-get update
+RUN apt-get install curl libpq-dev postgresql-client nodejs yarnpkg -y
+
+# Set working directory
+WORKDIR /home/indexer
+
+# Copy source code
+COPY . .
+
+# Install indexer
+RUN pip3 install poetry==1.6.1
```

Comment:
note: this is an adjustment that i needed to make because previously i was using the `python3-poetry` from debian package manager, and that version of poetry did not support the export command in line 16 below

Reply:
5fab0dbd67735c6a8fb7be2759278507d0522929